vpr/src/base/oo_tcl.h

```diff
@@ -0,0 +1,348 @@
+#ifndef OO_TCL_H
+#define OO_TCL_H
+
+#include <string>
+#include <list>
+#include <memory>
+
+#include <tcl/tcl.h>
+
+/**
+ * \file
```

Comment:
All the other doxygen comments have `@` instead of `\`, we should stick with the rest I think

Reply:
Some subprojects of VTR use `\`, but indeed, VPR uses `@`.
Fixed.